dags.yaml

```diff
@@ -1487,6 +1487,24 @@ bqetl_glam_refresh_aggregates:
     - impact/tier_2
     - repo/bigquery-etl
 
+  bqetl_glam_refresh_aggregates_fog_release:
```

Comment:
I think there's an extra indent in this block

Reply:
thanks, that's what's failing ci